src/pages/Home.tsx

```diff
@@ -1,8 +1,16 @@
 import { Box, Toolbar, Typography } from '@mui/material';
```

Comment:
Should this component be renamed to `Dashboard`?

Reply:
I think so eventually.